configs/keycloak_bridge.yml

```diff
@@ -104,6 +104,20 @@ db-config-ro-migration: false
 db-config-ro-migration-version: 0.1
 db-config-ro-connection-check: true
 
+# DB Users RW
+db-users-rw-host-port: 172.17.0.2:3306
+db-users-rw-username: bridge
+db-users-rw-password: bridge-password
+db-users-rw-database: configuration
```

Comment:
Are you using the same database than for "config" on purpose?

Reply:
Fixed: users